composer.json

```diff
@@ -42,6 +42,9 @@
             "drupal/ajax_comments": {
                 "Ajax not working when using non-default view mode": "https://www.drupal.org/files/issues/2023-01-30/ajax_comments-ajax_non_default_view_mode-2896916-beta5-60.patch"
             },
+            "drupal/color" : {
+                "Issue #1236098: Notice: Undefined index: 'base' in _color_rewrite_stylesheet()": "https://www.drupal.org/files/issues/undefined-index-in-_color_rewrite_stylesheet-1236098-37.patch"
```

Comment:
This patch is still written for core, we need to change it for contrib (see paths for example)

![Screenshot 2023-10-29 at 11 18 04](https://github.com/goalgorilla/open_social/assets/16667281/9596cb2e-59f5-4845-8899-900f69f8aaf3)

Reply:
OK